### PR TITLE
Issue 4414 - disk monitoring - prevent division by zero crash

### PR DESCRIPTION
**Pull request by bsimonova**
Description:
Added a test to check DS will not crash when division by zero
occurs in disk monitoring. Also fixed a description in compact_test.py
because it was causing errors in Polarion.

Relates: https://github.com/389ds/389-ds-base/issues/4414

Reviewed by: ???

### CODE REVIEW

File: dirsrvtests/tests/suites/disk_monitoring/disk_monitoring_divide_test.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        1. Success
+        2. Success
+        3. Success
+    """
+
+    standalone = topology_st.standalone
+
+    log.info('Check search works before changing the nsslapd-auditlog attribute')
+    try:
+        DSLdapObjects(topology_st.standalone, basedn='cn=disk space,cn=monitor').filter("(objectclass=*)", scope=0)
```
Comment (droideck) [inline dirsrvtests/tests/suites/disk_monitoring/disk_monitoring_divide_test.py:75]: We have an object that can be used for that - https://github.com/389ds/389-ds-base/blob/master/src/lib389/lib389/monitor.py#L353 - MonitorDiskSpace.

Could you please try if it'll work with '**filter**' function?

**Review by droideck — CHANGES_REQUESTED** [review]

File: dirsrvtests/tests/suites/disk_monitoring/disk_monitoring_divide_test.py (hunk 1 above)
Comment (bsimonova) [reply dirsrvtests/tests/suites/disk_monitoring/disk_monitoring_divide_test.py:75]: I tried the suggested object but it doesn't have filter attribute.

Comment (droideck) [reply dirsrvtests/tests/suites/disk_monitoring/disk_monitoring_divide_test.py:75]: Ah, right, MonitorDiskSpace is a singular object (and only DSLdapObjects has the filter method).

**Review by droideck — APPROVED** [review]
I think it's okay to use DSLdapObjects directly here as it's really a testing case and there's no need to add something like Monitor**s**DiskSpace.

You have my ack! Thank you!

**Pull request merged**